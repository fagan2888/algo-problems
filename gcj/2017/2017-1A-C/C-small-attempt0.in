100
11 5 16 5 0 0
3 1 3 2 2 0
3 1 3 2 1 0
2 1 5 1 1 1
99 2 2 8 0 0
98 1 96 49 0 1
99 1 92 48 0 1
9 1 79 3 2 2
23 7 94 11 8 1
74 1 56 16 1 14
67 1 76 36 1 11
16 5 61 8 4 2
98 1 92 48 0 1
92 1 19 23 0 0
95 1 45 49 0 2
92 1 94 45 0 1
90 1 95 46 0 1
98 1 95 48 0 1
93 1 48 45 0 2
91 1 93 45 0 1
90 1 96 45 0 1
93 1 97 45 0 1
92 1 99 46 0 1
93 1 99 47 0 1
21 1 5 26 4 4
98 1 93 48 0 1
73 1 89 44 4 42
97 1 98 47 0 1
88 1 75 44 3 17
77 1 24 34 3 32
91 1 97 45 0 1
97 1 47 48 0 2
90 1 31 47 0 3
96 1 91 48 0 1
100 1 99 51 0 1
26 1 60 13 3 2
75 5 51 41 3 7
86 1 85 16 1 2
97 1 31 48 0 3
28 5 50 7 0 0
51 21 78 29 10 21
91 1 91 44 0 1
90 1 92 44 0 1
100 1 30 51 0 3
98 1 23 53 0 4
88 7 48 60 0 0
71 1 38 40 1 9
96 1 96 48 0 1
78 1 13 91 3 20
92 1 90 45 0 1
97 1 96 48 0 1
93 1 48 47 0 2
97 1 22 48 0 4
91 1 93 46 0 1
96 1 93 49 0 1
95 1 93 46 0 1
30 1 7 89 1 2
100 1 24 52 0 4
37 1 93 55 2 31
97 1 100 48 0 1
17 1 4 9 4 4
97 1 91 48 0 1
90 1 94 44 0 1
30 65 77 89 1 12
9 1 33 5 3 1
99 1 97 50 0 1
36 41 85 76 28 49
99 1 31 50 0 3
99 1 99 48 0 1
92 1 91 41 3 5
90 1 97 44 0 1
93 1 30 49 0 3
46 1 93 21 3 4
100 1 100 51 0 1
99 1 32 49 0 3
97 1 95 48 0 1
96 1 97 47 0 1
95 4 23 34 0 0
19 3 12 29 2 17
92 1 25 46 0 4
100 1 98 49 0 1
97 1 23 52 0 4
94 1 91 46 0 1
90 1 92 45 0 1
62 1 33 31 3 3
67 1 85 25 1 4
97 1 30 51 0 3
100 1 24 49 0 4
92 1 94 47 0 1
42 1 51 22 3 3
56 10 85 25 2 2
94 1 45 47 0 2
98 1 95 49 0 1
92 1 24 49 0 4
80 15 55 89 33 87
95 1 47 49 0 2
100 1 99 50 0 1
99 1 47 51 0 2
4 1 93 33 1 17
88 1 64 37 4 6
