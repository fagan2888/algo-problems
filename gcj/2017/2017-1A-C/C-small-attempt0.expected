Case #1: 5
Case #2: 2
Case #3: IMPOSSIBLE
Case #4: 5
Case #5: 1
Case #6: 168
Case #7: 162
Case #8: 15
Case #9: 11
Case #10: 15
Case #11: 21
Case #12: 11
Case #13: 162
Case #14: 27
Case #15: 80
Case #16: 159
Case #17: 163
Case #18: 165
Case #19: 80
Case #20: 158
Case #21: 162
Case #22: 162
Case #23: 166
Case #24: 168
Case #25: IMPOSSIBLE
Case #26: 163
Case #27: 11
Case #28: 166
Case #29: 13
Case #30: 7
Case #31: 162
Case #32: 81
Case #33: 53
Case #34: 161
Case #35: 174
Case #36: 16
Case #37: 13
Case #38: 21
Case #39: 53
Case #40: 13
Case #41: 5
Case #42: 154
Case #43: 156
Case #44: 53
Case #45: 42
Case #46: IMPOSSIBLE
Case #47: 17
Case #48: 166
Case #49: IMPOSSIBLE
Case #50: 155
Case #51: 166
Case #52: 82
Case #53: 38
Case #54: 160
Case #55: 165
Case #56: 159
Case #57: IMPOSSIBLE
Case #58: 42
Case #59: 16
Case #60: 170
Case #61: 2
Case #62: 161
Case #63: 158
Case #64: IMPOSSIBLE
Case #65: 13
Case #66: 171
Case #67: 5
Case #68: 54
Case #69: 169
Case #70: 19
Case #71: 161
Case #72: 53
Case #73: 17
Case #74: 175
Case #75: 55
Case #76: 165
Case #77: 165
Case #78: 9
Case #79: 6
Case #80: 40
Case #81: 169
Case #82: 41
Case #83: 157
Case #84: 158
Case #85: 13
Case #86: 25
Case #87: 54
Case #88: 40
Case #89: 163
Case #90: 15
Case #91: 15
Case #92: 78
Case #93: 167
Case #94: 41
Case #95: 4
Case #96: 82
Case #97: 172
Case #98: 84
Case #99: IMPOSSIBLE
Case #100: 13
